lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -60,9 +60,8 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
       {elevator_widget_instance, header_footer_variant} =
         if is_nil(current_elevator_closure) do
           {%ElevatorClosuresList{
-             in_station_closures: in_station_closures,
```

Comment:
Since `in_station_closures` is now only used in the creation of `current_elevator_closure`, that logic could be merged into a single pipeline.